src/applications/personalization/profile360/actions/updaters.js

```diff
@@ -43,22 +44,14 @@ export function getTransactionStatus(transaction, fieldName) {
 
       // Check to see if the transaction is finished
       if (response.data.attributes.transactionStatus === VET360_CONSTANTS.TRANSACTION_STATUS.COMPLETED_SUCCESS) {
-        const currentState = getState();
-        const newValue = currentState.vaProfile.formFields[fieldName].value;
-
-        // Remove the transaction with a delay for effect
-        setTimeout(() => {
-          dispatch({
-            type: VET360_TRANSACTION_FINISHED,
-            fieldName
-          });
-        }, 3000);
-
-        // Update the property on the FE
+
+        // Refresh the profile object
+        await dispatch(getProfile());
+
+        // Remove the transaction from the VA Profile
```

Comment:
This ultimately checks the status for a Vet360 transaction and dispatches the getUser action creator if the transaction is completed.